src/libs/ReportScrollManager/index.native.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const flatListRef = React.createRef();
```

Comment:
Also, let's gather all exports together into one statement. Something like this:

```javascript
const flatListRef = React.createRef();
function scrollToIndex() {
    ...
}

export {
    flatListRef,
    scrollToIndex,
};
```